tensorflow/core/lib/random/random_distributions_utils.h

```diff
@@ -22,6 +22,10 @@ limitations under the License.
 
 #include "tensorflow/core/lib/random/philox_random.h"
 
+#ifndef M_PI
```

Comment:
Why is this change needed?

Reply:
M_PI is not defined in C standard. M_PI is currently used only in this file.

Reply:
@MeghnaNatraj This is not the correct way to introduce math constant, you are mis-clarified!

@mattn  See MSVC document of [Math Constants](https://docs.microsoft.com/en-us/cpp/c-runtime-library/math-constants?view=msvc-170) to see how to do it in sane way.